### PR TITLE
Exposing config type once resolved and running

### PR DESCRIPTION
**Pull request by WebReflection**
## Description

This MR fixes https://github.com/pyscript/pyscript/issues/2174 by exposing a `type` in the already handy `config` we provide.

## Changes

  * add a check for the *MicroPython* interpreter/runtime
  * set a `config["type"]`
  * test everything is working as expected

## Checklist

-   [x] All tests pass locally
-   [ ] I have updated `CHANGELOG.md`
-   [ ] I have created documentation for this(if applicable)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

### CODE REVIEW

File: pyscript.core/src/stdlib/pyscript/magic_js.py [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
 config = json.loads(globalThis.JSON.stringify(_config))
 
+if "MicroPython" in sys.version:
```
Comment (ntoll) [inline pyscript.core/src/stdlib/pyscript/magic_js.py:13]: I usually normalise to: `if "micropython" in sys.version.lower()` But I don't think it makes much difference.

**Review by ntoll — APPROVED** [review]

File: pyscript.core/src/stdlib/pyscript/magic_js.py (hunk 1 above)
Comment (WebReflection) [reply pyscript.core/src/stdlib/pyscript/magic_js.py:13]: I believe they really want the MicroPython brand to be **MicroPython** but I don't mind applying that change ... alternatively I could use other try/catches or pass the type from the page but that is actually pretty convoluted due workers VS main VS config in PyScript VS Polyscript and so on ... I think this was the fastest and easy way around.

Comment (ntoll) [reply pyscript.core/src/stdlib/pyscript/magic_js.py:13]: Yeah... it's also the smallest possible change. We can refine later if there are problems / feature requests. Was just mentioning it as a sort of "out of interest..." point. :+1:

**Review by ntoll — APPROVED** [review]

**Comment by ntoll** [comment]
Also, apologies, I thought I had marked this as approved when I first commented. My bad.

**Pull request merged**